### ISSUE TITLE
Ability for a donor to pledge funds and for that pledge to be recorded like aid packages as institutional donors cannot pay by credit card. 

### ISSUE THREAD

**Issue by nshamit** [LSFLK/MedicinesforLK#364]
Ability for a donor to pledge funds instead of pledging towards an aid package, and for that pledge to be recorded like aid packages as institutional donors cannot pay by credit card in the give.asia site.

**Comment by jayasanka-sack** [comment]
According to the current implementation, it's impossible to create an aid package without medicine. Amending the flow would break the structure and validations. I think it's better to come up with a different approach. 

@manuranga @NipunaMarcus please let us know your thoughts on this.

**Comment by NipunaMarcus** [comment]
Yes, I agree with @jayasanka-sack ... Current flow or the discussed flow was to create aid packages with need packages... If we are to create an aid package without needs, we can introduce another type of package... Something like "DonationPackage" .... Also this will involve a few DB changes and package creation API changes.

**Comment by manuranga** [comment]
Lets add 'kind' to medical item that says its a donation and set a fake amount like 1000000. Elegant way to do it would be use a different table l, but that will require significant changes.
